scripts/Thaumic-Horizons.zs

```diff
@@ -0,0 +1,82 @@
+// --- Created By DreamMasterXXL --- 
+
+
+
+// --- Removing Recipes ---
+
+
+
+// --- Planar Conduit
+mods.thaumcraft.Arcane.removeRecipe(<ThaumicHorizons:planarConduit>);
+
+// --- Transduction Amplifier
+mods.thaumcraft.Arcane.removeRecipe(<ThaumicHorizons:transductionAmplifier>);
+
+// --- Vortex Attenuation
+mods.thaumcraft.Infusion.removeRecipe(<ThaumicHorizons:vortexStabilizer>);
+
+// --- Teleological Recombinator
+mods.thaumcraft.Infusion.removeRecipe(<ThaumicHorizons:recombinator>);
+
+
+
+
+// --- Adding Recipes ---
+
+
+
+// --- Planar Conduit
```

Comment:
These general thaumic horizons nerfs really shouldn't happen. This gates the majority of thaumic horizons stuff to tungstensteel coils (for enderium). Most of this stuff is earlygame magic things, like slightly increasing the power of an energised node, or temporarily making a hungry node safe. This also nerfs the entirety of void crafting, which didn't need a nerf.